src/payment.js

```diff
@@ -414,13 +387,13 @@ Payment.prebuild = function (absoluteFee) {
         payment.extraFeeConsumption = 0;
       }
 
-      // compute absolute fee bounds for 1,2,3,4,5,6 block confirmations
+      // compute absolute fee bounds for slow, regular, priority block confirmations
       var toAbsoluteFee = function (e) {
-        var c = Transaction.filterUsableCoins(payment.coins, e.fee);
-        var s = Transaction.selectCoins(c, amounts, e.fee, false);
+        var c = Transaction.filterUsableCoins(payment.coins, e[1] * 1000);
+        var s = Transaction.selectCoins(c, amounts, e[1] * 1000, false);
         return s.fee;
       };
-      payment.absoluteFeeBounds = payment.fees.estimate.map(toAbsoluteFee);
+      payment.absoluteFeeBounds = Object.entries(payment.fees).map(toAbsoluteFee);
 
       // estimation of confirmation in number of blocks
       payment.confEstimation = Transaction.confirmationEstimation(payment.absoluteFeeBounds, payment.finalFee);
```

Comment:
`confEstimation` doesn't seem relevant anymore, could you get rid of that + the code it uses.